src/WeatherData.cpp

```diff
@@ -0,0 +1,139 @@
+#include "WeatherData.hpp"
+#include "merc.h"
+
+#include <fmt/format.h>
+
+using namespace fmt::literals;
+
+WeatherData weather_info;
+
+namespace {
+
+std::string_view to_string(WeatherData::Sky sky) {
+    switch (sky) {
+    case WeatherData::Sky::Cloudless: return "cloudless";
```

Comment:
At some point we should add some crazy weather like snow, hail, tornados :)

Reply:
Another thing we could do is make weather affect gameplay e.g. raining might reduce your 'scan' range. Cold/wet conditions might cause you to experience reduced dexterity etc.

Reply:
Nice ideas! Absolutely!